### ISSUE TITLE
AM 2.6.1 Crash

### ISSUE THREAD

**Issue by NSTAdventure** [MuntashirAkon/AppManager#471]
- [x] I know what my device, OS and App Manager versions are
- [x] I know how to take logs
- [ ] I know how to reproduce the issue which may not be specific to my device

**Describe the bug**
App manager suddenly crash

**To Reproduce**
Steps to reproduce the behaviour:
1. enable ADB over TCP
2. launch AM v2.6.1
3. AM crash

**Expected behavior**
not crashing

**Screenshots**

**Crash logs**
```
java.lang.ArrayIndexOutOfBoundsException: length=5; index=-1
    at io.github.muntashirakon.AppManager.settings.MainPreferences.onCreatePreferences(MainPreferences.java:177)
    at androidx.preference.PreferenceFragmentCompat.onCreate(PreferenceFragmentCompat.java:160)
    at androidx.fragment.app.Fragment.performCreate(Fragment.java:2949)
    at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:475)
    at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:278)
    at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2189)
    at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2100)
    at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:2002)
    at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3138)
    at androidx.fragment.app.FragmentManager.dispatchActivityCreated(FragmentManager.java:3072)
    at androidx.fragment.app.FragmentController.dispatchActivityCreated(FragmentController.java:251)
    at androidx.fragment.app.FragmentActivity.onStart(FragmentActivity.java:501)
    at androidx.appcompat.app.AppCompatActivity.onStart(AppCompatActivity.java:246)
    at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1446)
    at android.app.Activity.performStart(Activity.java:7990)
    at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3535)
    at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:237)
    at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:217)
    at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:189)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:113)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2213)
    at android.os.Handler.dispatchMessage(Handler.java:107)
    at android.os.Looper.loop(Looper.java:238)
    at android.app.ActivityThread.main(ActivityThread.java:7853)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:984)

Device Info:
App version: 2.6.1
App version code: 387
Android build version: 1608537052
Android release version: 10
Android SDK version: 29
Android build ID: RMX2030EX_11_C.67
Device brand: realme
Device manufacturer: realme
Device name: RMX2030
Device model: RMX2030
Device product name: RMX2030
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: id-ID
In-App Language: auto
Mode: adb
```

**Device info**
 - Device: Realme 5i
 - OS Version: Android 10 Realme UI 
 - App Manager Version: 2.6.1
 - Mode: ADB

**Additional context**
Onboard Page Empty Reload Won't Help You

**Comment by MuntashirAkon** [comment]
This should only occur if you had set ADB over TCP mode in a previous version of AM. The only way to fix it would be to downgrade to the previous version or clear data. This will be fixed in the next version.

**Comment by MuntashirAkon** [comment]
Fixed in 46ec1868
